yt_dlp/extractor/prankcast.py

```diff
@@ -42,7 +62,7 @@ def _real_extract(self, url):
             'uploader': uploader,
             'channel_id': json_info.get('user_id'),
             'duration': try_call(lambda: parse_iso8601(json_info['end_date']) - start_date),
-            'cast': list(filter(None, [uploader] + traverse_obj(guests_json, (..., 'name')))),
+            'cast': list(filter(None, [uploader] + (try_call(lambda: traverse_obj(guests_json, (..., 'name'))) or []))),
```

Comment:
`traverse_obj` should never throw and supports a default
```suggestion
            'cast': list(filter(None, [uploader] + traverse_obj(guests_json, (..., 'name'), default=[]))),
```

Reply:
Even default isnt needed here. What's wrong with the old code?